Scripts/Core/EditorVR.Menus.cs

```diff
@@ -99,7 +99,7 @@ internal void UpdateMenuVisibilityNearWorkspaces()
 						}
 
 						if (m_WorkspaceModule == null)
```

Comment:
As we discussed, go ahead and get rid of the module caching across the board, except where absolutely needed (e.g. m_Interfaces)